Dockerfile

```diff
@@ -12,4 +12,4 @@ RUN git config --global user.name "readme-bot"
 RUN git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
 
 ADD sources/* /waka-readme-stats/
-ENTRYPOINT cd /waka-readme-stats/ && python3 main.py
+ENTRYPOINT ["cd", "/waka-readme-stats/", "&&", "python3 main.py"]

```

Comment:
To best of my knowledge, this won't work. List-like syntax is only available for commands consisting of one executable and multiple arguments. Here, what happens is a shell invocation, with two different commands, chained by "&&". That is why single string entrypoint definition style is the only available choice here.